src/main/webapp/babel-src/feedBackBox.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import TextField from 'material-ui/TextField';
+import Snackbar from 'material-ui/Snackbar';
+import {FEEDBACK_CHAR_LIMIT, FEEDBACK_SNACKBAR_AUTO_HIDE_DURATION, UI_STRINGS, INLINE_STYLES} from './util';
+
+export class FeedBackBox extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            charCtr : 0,
+            errorMsg : '',
+            feedbackMsg: '',
+            sendSuccess : false,
+            sendError : false

```

Comment:
Having a sendError flag on top of sendSuccess is redundant and more error-prone IMO. We can just check if sendSuccess is false to achieve the same result. This concern also reinforces the idea that we should probably only create 1 snackbar component which can show either the success or failure message. Not necessary to fix if you don't feel the need to but I thought it was important enough to mention.